src/E225B0.c

```diff
@@ -1,7 +1,203 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "E225B0", func_802B7000_E225B0);
+void func_8013AA9C(s32, s32, s32, s32, s32, s32, s32); // extern
```

Comment:
I'd move this function decl down to be with the other one (and the comment can be removed)